### PR TITLE
feat: add namespace configuration for secret-store-csi-driver helm subchart

### PR DESCRIPTION
**Pull request by bhanuprakash-1**
Adding configurability of namespace for secret-store-csi-driver as asked in this open issue: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/575

**What type of PR is this?**
feature

**What this PR does / why we need it**:
There is a requirement of configurability for namespace of this driver and the charts which are dependent on this. Refer to this issue: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/575. 

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

### CODE REVIEW

File: manifest_staging/charts/secrets-store-csi-driver/README.md [hunk 1]
```diff
@@ -107,3 +107,4 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `providerHealthCheckInterval`           | Provider healthcheck interval duration                                                                                                                                         | `2m`                                                    |
 | `imagePullSecrets`                      | One or more secrets to be used when pulling images                                                                                                                             | `""`                                                    |
 | `tokenRequests`                         | Token requests configuration for the csi driver. Refer to [doc](https://kubernetes-csi.github.io/docs/token-requests.html) for more info. Supported only for Kubernetes v1.20+ | `""`                                                    |
+| `namespace`                             | If given, installs the driver in the given namespace                                               | `""`                                             |
```
Comment (nilekhc) [inline manifest_staging/charts/secrets-store-csi-driver/README.md:110]: Could you add empty entry in values.yaml?

**Comment by nilekhc** [comment]
Thanks for the PR @bhanuprakash-1. Could you sing CLA? It's required to accept contributions. https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by bhanuprakash-1** [comment]
> Thanks for the PR @bhanuprakash-1. Could you sing CLA? It's required to accept contributions. https://github.com/kubernetes/community/blob/master/CLA.md

I have done the required(signed the documente, but it still shows missing CLA Authorization.

File: manifest_staging/charts/secrets-store-csi-driver/README.md (hunk 1 above)
Comment (bhanuprakash-1) [reply manifest_staging/charts/secrets-store-csi-driver/README.md:110]: Have added. Thanks.

**Comment by aramase** [comment]
/retitle feat: add namespace configuration for secret-store-csi-driver helm subchart

@bhanuprakash-1 could you squash the commits? Hopefully, the CLA issue will be resolved with the new push. If it still doesn't work, you can try reaching out to them [here](https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting).

**Review by aramase — COMMENTED** [review]
@bhanuprakash-1 Here is more detail on squashing commits: https://www.kubernetes.dev/docs/guide/github-workflow/#squash-commits

**Comment by aramase** [comment]
@bhanuprakash-1 Please see https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/1015#issuecomment-1209704930 for CLA.

I'm closing this PR now due to inactivity. Please feel free to reopen if it's still being worked on. 

/close

**Pull request closed without merging**